src/v2/view-builder/views/phone/ChallengeAuthenticatorPhoneView.js

```diff
@@ -107,7 +109,8 @@ export default BaseAuthenticatorView.extend({
         'type': 'string',
       },
       phoneNumber: {
-        'value': profile.phoneNumber,
+        // TODO: Clean-up post M1 when phoneNumber is available in all cases - enrollment & verification.
```

Comment:
create ticket for todo and add `m2` label

Reply:
https://oktainc.atlassian.net/browse/OKTA-312428